playbooks/provisioner/openstack/main.yaml

```diff
@@ -4,3 +4,5 @@
   roles:
       - { role: openstack/neutron, action: setup }
       - { role: openstack/nova, action: setup }
+
+- include: plugins.yml
```

Comment:
if it's not Openstack specific, it needs to be as a separate "ad-hoc" playbook.
Much like how we do "scale", we don't put it as part of the osp-d deployment playbook, we have a separate playbook for it.
inside the Openstack role, you can have: 
- include: bmc.yaml
  when: provisioner.nodes.bmc is defined (much like how the virthost in osp-d playbook works, you can't remove it as it dependent whether we have a virthost or not